packages/appstore/src/components/main-title-bar/asset-summary.tsx

```diff
@@ -52,6 +61,17 @@ const AssetSummary = observer(() => {
                         zIndex={9999}
                         is_bubble_hover_enabled
                     >
+                        <h1>new</h1>
```

Comment:
this is temp. just to check the current calculations with the old one. will remove the old one once I'm done with the refactoring

Reply:
Can you elaborate more on the plan, I don't think I can fully understand what will happen next.

Reply:
@matin-deriv so if the QA team want to compare the test link with the production, the difference in exchange rates (for production and test-link) results in varying numbers and the QA team definitely going to send back the card to `in progress` . 
I checked this with the QA and they told me in that case let's keep the old code for total-asset so they can check the numbers and once they're done with the tests I can just remove the old ones. this is only happening for tradershub. for p2p and cashier we don't need this.

Reply:
I could go with adding console log but since we have multiple re-renderings there would be lots of console logs which was harder to track

Reply:
@shayan-deriv we need to make sure to remove this after qa checkin is don so don't merge this PR without "qa related code"
@shayan-deriv please add a comment above the code

Reply:
done ✅  @mahdiyeh-deriv